DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -16,9 +16,10 @@
   <class name="pat::Lepton<reco::BaseTau>" />
 
   <!-- PAT Objects, and embedded data  -->
-  <class name="pat::Electron"  ClassVersion="29">
-   <version ClassVersion="30" checksum="3949366163"/>
-   <version ClassVersion="29" checksum="1784986402"/>
+  <class name="pat::Electron"  ClassVersion="31">
+   <version ClassVersion="31" checksum="3508125821"/>
+   <version ClassVersion="30" checksum="1784986402"/>
+   <version ClassVersion="29" checksum="1881133053"/>
```

Comment:
(now that David mentioned it)
is this set of version-checksum pairs actually correct?
As I understand now, the existing (and correct, in a sense of some data already created with it) "checksum-version" pair should not be changed.
Here, the checksum from  "29-1784986402" is now in  "30-1784986402" and the checksum from "30-3949366163" became is now gone.
If the head 
is correct and these reassignments aren't a result of some bugfix, then the 
```
<version ClassVersion="30" checksum="3949366163"/>
<version ClassVersion="29" checksum="1784986402"/>
```
should stay
and 
``` <version ClassVersion="29" checksum="1881133053"/>  ```  should go away (or become 31 and the current 31 become 32)